janusgraph-core/src/main/java/org/janusgraph/graphdb/database/management/ManagementSystem.java

```diff
@@ -946,13 +950,15 @@ private UpdateStatusTrigger(StandardJanusGraph graph, JanusGraphSchemaVertex ver
             this.graph = graph;
             this.schemaVertexId = vertex.longId();
             this.newStatus = newStatus;
-            this.propertyKeys = Sets.newHashSet(Iterables.transform(keys, new Function<PropertyKey, Long>() {
+            Iterable<? extends Long> it = Iterables.transform(keys, new Function<PropertyKey, Long>() {
```

Comment:
Good point, as I noted in this PR I didn't go through all the code because it is hard to move all guava usages in once and I was planning to move all guava usages in chunks but I will move this part to stream api as you noted here.